nextcord/application_command.py

```diff
@@ -111,6 +117,48 @@ def _cog_special_method(func: FuncT) -> FuncT:
     return func
 
 
+class InteractionDefault:
+    # TODO: fix docstring lol
+    """A class that makes all thing default.
```

Comment:
reword this, what is "thing"

Reply:
i mean, i don't have any idea to write

Reply:
`thing` still isnt an acceptable description, use plain english - this is a default value setter for application command arguments, so describe it that way